src/Command/PingStaleIssuesCommand.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Command;
+
+use App\Api\Issue\IssueApi;
+use App\Api\Issue\IssueType;
+use App\Api\Label\LabelApi;
+use App\Entity\Task;
+use App\Service\RepositoryProvider;
+use App\Service\StaleIssueCommentGenerator;
+use App\Service\TaskScheduler;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Close issues not been updated in a long while.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class PingStaleIssuesCommand extends Command
+{
+    public const STALE_IF_NOT_UPDATED_SINCE = '-12months';
+    public const MESSAGE_TWO_AFTER = '+2weeks';
+    public const MESSAGE_THREE_AND_CLOSE_AFTER = '+2weeks';
+
+    protected static $defaultName = 'app:issue:ping-stale';
+
+    private $repositoryProvider;
+    private $issueApi;
+    private $scheduler;
+    private $commentGenerator;
+    private $labelApi;
+
+    public function __construct(RepositoryProvider $repositoryProvider, IssueApi $issueApi, TaskScheduler $scheduler, StaleIssueCommentGenerator $commentGenerator, LabelApi $labelApi)
+    {
+        parent::__construct();
+        $this->repositoryProvider = $repositoryProvider;
+        $this->issueApi = $issueApi;
+        $this->scheduler = $scheduler;
+        $this->commentGenerator = $commentGenerator;
+        $this->labelApi = $labelApi;
+    }
+
+    protected function configure()
+    {
+        $this->addArgument('repository', InputArgument::REQUIRED, 'The full name to the repository, eg symfony/symfony-docs');
```

Comment:
Sorry, but could we add a `--dry-run` option? And also write output in the command? For example:

> Adding label and comment to issue #555 on symfony/symfony

And, of course, the `--dry-run` would make no changes. It would make me a bit more comfortable, as we could deploy, then run the command manually with `--dry-run` and make sure there are no surprises (like WAY too many issues being commented on, due to some bug).

Thanks!

Reply:
The ´--dry-run´ option is added. 

It will show you 100 issues on Symfony/symfony on the first run. We do not use pagination because it isn't really needed. 

Here is a link to the search: https://github.com/symfony/symfony/issues?q=is%3Aissue+-linked%3Apr+-label%3A%22Keep+open%22+is%3Aopen+updated%3A%3C2019-12-18